sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationClient.java

```diff
@@ -137,7 +137,7 @@ public ConfigurationSetting addSetting(ConfigurationSetting setting) {
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
     public ConfigurationSetting setSetting(String key, String value) {
-        return setSetting(new ConfigurationSetting().key(key).value(value), Context.NONE).value();
+        return setSetting(new ConfigurationSetting().key(key).value(value), Context.NONE).getValue();
```

Comment:
Shouldn't the `key()` and `value()` method also be renamed? Same in async client too.
```
        return setSetting(new ConfigurationSetting().setKey(key).setValue(value), Context.NONE).getValue();
```

Reply:
Yep, once app configuration is refactored.